### PR TITLE
release notes for 193

### PR DESCRIPTION
**Pull request by AlanRoth**

### CODE REVIEW

File: release-notes/release-notes-193.adoc [hunk 1]
```diff
@@ -0,0 +1,115 @@
...
+-   [https://github.com/payara/Payara/pull/4124[PAYARA-4034]] - Cleanup of sonar warnings for common/amx-core
+-   [https://github.com/payara/Payara/pull/4150[PAYARA-4064]] - Stabilize creation of default domain in build
+-   [https://github.com/payara/Payara/pull/4135[PAYARA-4065]] - Reinitialize PayaraExecutorService on embedded server restart
+-   [https://github.com/payara/Payara/pull/4151[PAYARA-4073]] - Use correct constructor of NotAuthorizedException in RolesAllowedRequestFilter
+
+## Security
+
+-   [https://github.com/payara/Payara/pull/4004[PAYARA-3881]] - CVE-2019-12086 Jackson Databind upgrade
+
+## Bug
```
Comment (jbee) [inline release-notes/release-notes-193.adoc]: Shouldn't this be _Bug fixes_ ?

Comment (Pandrex247) [reply release-notes/release-notes-193.adoc]: Too slow ;)

Comment (AlanRoth) [reply release-notes/release-notes-193.adoc]: Yes, protected branches mean I cant do a quick commit to fix it, its in a seperate PR

Comment (AlanRoth) [reply release-notes/release-notes-193.adoc]: Even though I created this branch

Comment (jbee) [reply release-notes/release-notes-193.adoc]: Its in...

**Pull request merged**